docs/installation/docker.md

```
@@ -12,44 +12,93 @@ permalink: /installation/docker/
-  -e DATABASE_URL=postgresql://skylite:password@skylite-ux-db:5432/skylite \
-  -e NUXT_PUBLIC_TZ=America/Chicago \
-  -e NUXT_PUBLIC_LOG_LEVEL=warn \
   -p 3000:3000 \
-  wetzel402/skylite-ux:beta
+  -v ~/skylite-data:/data \
+  -e NUXT_PUBLIC_TZ=America/Chicago \
+  --name skylite-ux \
+  y3knik/skylite-ux:beta
 ```
 
+Then open http://localhost:3000 in your browser.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)